x/pool-incentives/types/gov.go

```diff
@@ -39,7 +39,7 @@ func (p *ReplacePoolIncentivesProposal) GetDescription() string { return p.Descr
 func (p *ReplacePoolIncentivesProposal) ProposalRoute() string { return RouterKey }
 
 func (p *ReplacePoolIncentivesProposal) ProposalType() string {
-	return ProposalTypeUpdatePoolIncentives
+	return ProposalTypeReplacePoolIncentives
 }
```

Comment:
ooof, would this be state breaking to backport? (I don't have a sense for where ProposalType gets used)

Reply:
I think they're used in routing

Reply:
This is used in x/gov routing when executing the proposal's handler.

Reply:
If it's only used for routing, I assume it's not going to be state-breaking to backport?

Reply:
I think If we have a ProposalTypeReplacePoolIncentives submited, this will be state-breaking